docs/docs/starlark-reference/service-config.md

```diff
@@ -109,6 +124,39 @@ If you are trying to use a more complex versions of `cmd` and are running into i
 set the `entrypoint` to `["/bin/sh", "-c"]` and then set the `cmd` to the command as you would type it in your shell. For example, `cmd = ["echo foo | grep foo"]`
 :::
 
+:::tip Example of a rendered label
+If you have defined the pod label key:value pair in `ServiceConfig` to be:
+
+```py
+config = ServiceConfig(
```

Comment:
This section is formatted weirdly - can you fix?

Reply:
I'll do it